### PR TITLE
add helpdesk user with same perms as admin

### PR DESCRIPTION
**Pull request by macrael**
## Summary

We want our helpdesk to have helpdesk user types in IDM, this adds them to the app with the same permissions as admin users.

#### Test cases covered

Only one basic test was added so far for the parsing. 

## QA guidance

there should be a new helpdesk@example.com user you can log in with on the review app and confirm that you can see the same stuff an admin can see

### CODE REVIEW

File: services/app-api/src/resolvers/healthPlanPackage/fetchHealthPlanPackage.ts [hunk 1]
```diff
@@ -58,7 +59,11 @@ export function fetchHealthPlanPackageResolver(
                     'user not authorized to fetch data from a different state'
                 )
             }
-        } else if (isCMSUser(context.user) || isAdminUser(context.user)) {
+        } else if (
+            isCMSUser(context.user) ||
+            isAdminUser(context.user) ||
+            isHelpdeskUser(context.user)
```
Comment (JasonLin0991) [inline services/app-api/src/resolvers/healthPlanPackage/fetchHealthPlanPackage.ts:65]: With this new type of user who cannot fetch a draft, we should change the error message `'CMS user not authorized to fetch a draft'` to something more generic.

File: services/app-web/src/localAuth/LocalLogin.tsx [hunk 2]
```diff
@@ -72,6 +73,13 @@ const localUsers: LocalUserType[] = [
         familyName: 'Coldstart',
         role: 'ADMIN_USER',
     },
+    {
+        id: 'user5',
```
Comment (JasonLin0991) [inline services/app-web/src/localAuth/LocalLogin.tsx:77]: We already have a user with the id `user5`
```suggestion
        id: 'user7',
```

File: services/app-web/src/pages/SubmissionSideNav/SubmissionSideNav.tsx [hunk 3]
```diff
@@ -103,15 +103,19 @@ export const SubmissionSideNav = () => {
 
     const isCMSUser = loggedInUser?.role === 'CMS_USER'
     const isAdminUser = loggedInUser?.role === 'ADMIN_USER'
+    const isHelpdeskUser = loggedInUser?.role === 'HELPDESK_USER'
 
     // CMS Users can't see DRAFT, it's an error
-    if (submissionStatus === 'DRAFT' && (isCMSUser || isAdminUser)) {
+    if (
+        submissionStatus === 'DRAFT' &&
+        (isCMSUser || isAdminUser || isHelpdeskUser)
```
Comment (JasonLin0991) [inline services/app-web/src/pages/SubmissionSideNav/SubmissionSideNav.tsx:111]: Curious if a help desk user would ever be asked for support on a draft submission.

**Review by JasonLin0991 — APPROVED** [review]
Looks good!

The only thing that would need to be fixed is the user id in `LocalLogin.tsx`

File: services/app-api/src/resolvers/healthPlanPackage/fetchHealthPlanPackage.ts (hunk 1 above)
Comment (JasonLin0991) [reply services/app-api/src/resolvers/healthPlanPackage/fetchHealthPlanPackage.ts:65]: Unless help desk users are considered CMS users?

File: services/app-web/src/pages/SubmissionSideNav/SubmissionSideNav.tsx (hunk 3 above)
Comment (macrael) [reply services/app-web/src/pages/SubmissionSideNav/SubmissionSideNav.tsx:111]: Yes, they would, I'm going to file a ticket to track that. We should also make sure they can't pose questions etc. The goal with this PR is to get some access to our HD users quickly.

**Pull request merged**